nav2_bt_navigator/behavior_trees/navigate_w_recovery_and_constant_replanning_and_if_path_becomes_invalid.xml

```diff
@@ -0,0 +1,46 @@
+<!--
+  This Behavior Tree replans the global path once every 15 seconds or if the path becomes invalid. It also has
+  recovery actions specific to planning / control as well as general system issues.
+  This will be continuous if a kinematically valid planner is selected.
+-->
+<root main_tree_to_execute="MainTree">
+  <BehaviorTree ID="MainTree">
+    <RecoveryNode number_of_retries="6" name="NavigateRecovery">
+      <PipelineSequence name="NavigateWithReplanning">
+        <RateController hz="2.0">
+          <RecoveryNode number_of_retries="1" name="ComputePathToPose">
+            <Fallback>
+              <ReactiveSequence>
+                <Inverter>
+                  <PathExpiringTimer seconds="15" path="{path}"/>
```

Comment:
I know its a last minute thing -- but from the feedback on Discourse, can we push this down to 10s (and update any appropriate documentation)? I think 15s might be a little much and its the middle ground between the 5s request made

Reply:
I marked where in docs the 10s update needs to be made, just 3 places